python/social_bot/envs/grocery_ground.py

```diff
@@ -75,15 +75,20 @@ def __init__(self,
                  fail_distance_thresh=3,
                  random_range=10.0,
                  random_goal=False,
+                 sparse_reward=False,
```

Comment:
Let's make the default sparse_reward to True because that is the original task, and Le is working on the task with sparse reward.

Reply:
done